test/apiv2/test-apiv2

```diff
@@ -255,6 +255,9 @@ function t() {
 
         for arg; do
             case "$arg" in
+                # TODO: this is just some hack to avoid adding `-d {}` to curl for endpoints where we really need an empty body
+                -)                curl_args+=(--disable);
+                                  shift;;
```

Comment:
@edsantiago PTAL, I guess you have a better idea how to address this. Basically I have to prevent the jsonify logic below. I need an empty body for the test to work.

Reply:
Well.... I give up. I tried some different ideas on changing how/when `-d {}` gets set, but everything failed in different ways. If this approach works out for you, I'm OK with it.

Reply:
OK, repushed with a bit more comments.